portal-ui/src/screens/Console/Storage/StoragePVCs.tsx

```diff
@@ -126,8 +136,11 @@ const StorageVolumes = ({
               width: 120,
             },
             {
-              label: "Volume",
-              elementKey: "volume",
+              label: "Tenant",
+              renderFullObject: true,
+              renderFunction: (record: any) => {
+                return `${record.namespace}/${record.tenant}`
+              },
```

Comment:
Function can be simplified:

```suggestion
              renderFunction: (record: any) => `${record.namespace}/${record.tenant}`,
```